src/main/scala/diplomacy/LazyModule.scala

```diff
@@ -184,6 +192,11 @@ sealed trait LazyModuleImpLike extends RawModule
 
 class LazyModuleImp(val wrapper: LazyModule) extends MultiIOModule with LazyModuleImpLike {
   val (auto, dangles) = instantiate()
+
+  if(! s"${this.getClass.getName}".contains("anon")){      
```

Comment:
Why delete anonymous LazyModuleImps? Why use the className, instead of the module name (which is guaranteed to be unique)?

Reply:
how will this interact with FIRRTL transforms that modify module /instance names after chisel elaboration?